tests/Client/TestRepo.php

```diff
@@ -62,15 +64,19 @@ public function fetchFile(string $fileName, int $maxBytes)
     }
 
     /**
-     * Sets the file for which a signature fail should be triggered.
+     * Sets a value in a file to be changed in ::fetchFile().
```

Comment:
I don't understand the new purpose of this method from these docs.

Reply:
The class is a very convoluted now. I will change so that this method changes the repo files directly instead of storing the change information for later.

Reply:
Changed